<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>부천시 공공테니스장 예약매크로</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image_button/tennis.qrc">
    <normaloff>:/image/tennis_icon.ico</normaloff>:/image/tennis_icon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(152, 206, 142);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: rgb(86, 101, 200);
	border-radius: 10px;
	border: 4px solid rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="browserBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>185</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(255, 255, 255);

}
QComboBox:hover {
	border: 2px solid rgb(110, 129, 255);
}
QComboBox:focus {
	border: 2px solid rgb(72, 144, 216);
}
QListView {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <item>
       <property name="text">
        <string>Chrome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Firefox</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Microsoft Edge</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="idEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>185</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover {
	border: 2px solid rgb(110, 129, 255);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
}</string>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pwEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>185</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover {
	border: 2px solid rgb(110, 129, 255);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QComboBox" name="courtBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>185</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(255, 255, 255);
}
QComboBox:hover {
	border: 2px solid rgb(110, 129, 255);
}
QComboBox:focus {
	border: 2px solid rgb(72, 144, 216);
}
QListView {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <item>
       <property name="text">
        <string>남부수자원테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>종합운동장테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>부천체육관(하드)테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>성주산체육공원테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>원미테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>부천실내테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>오정레포츠센터 테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>소사배수지테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>복사골테니스장</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>해그늘체육공원 테니스장</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="timeBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>185</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(255, 255, 255);
}
QComboBox:hover {
	border: 2px solid rgb(110, 129, 255);
}
QComboBox:focus {
	border: 2px solid rgb(72, 144, 216);
}
QListView {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <item>
       <property name="text">
        <string>06:00~08:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08:00~10:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10:00~12:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12:00~14:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14:00~16:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16:00~18:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18:00~20:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20:00~22:00</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="infoButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>185</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(249, 249, 249);
	border-radius: 10px;
}
QPushButton:hover {
	border: 2px solid rgb(110, 129, 255);
}</string>
      </property>
      <property name="text">
       <string>제출</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>185</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(255, 255, 255);
}
QLineEdit:hover {
	border: 2px solid rgb(110, 129, 255);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
}</string>
      </property>
      <property name="placeholderText">
       <string>날짜(예시: 20230101)</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="infoBrowser">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>331</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: rgb(86, 101, 200);
	border-radius: 10px;
	color: rgb(255, 255, 255);
	border: 4px solid rgb(255, 255, 255);
	padding-left: 5px;
}</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="progressBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>281</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: rgb(86, 101, 200);
	border-radius: 10px;
	color: rgb(255, 255, 255);
	border: 4px solid rgb(255, 255, 255);
	padding-left: 5px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chromeButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>540</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(249, 249, 249);
	border-radius: 10px;
}
QPushButton:hover {
	border: 2px solid rgb(189, 255, 176);
}</string>
     </property>
     <property name="text">
      <string>매크로 시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="firefoxButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>540</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(249, 249, 249);
	border-radius: 10px;
}
QPushButton:hover {
	border: 2px solid rgb(189, 255, 176);
}</string>
     </property>
     <property name="text">
      <string>매크로 시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edgeButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>540</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(249, 249, 249);
	border-radius: 10px;
}
QPushButton:hover {
	border: 2px solid rgb(189, 255, 176);
}</string>
     </property>
     <property name="text">
      <string>매크로 시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>506</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(152, 206, 142);
	border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image_button/tennis.qrc">
       <normaloff>:/image/firefox_logo.png</normaloff>:/image/firefox_logo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>506</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(152, 206, 142);
	border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image_button/tennis.qrc">
       <normaloff>:/image/edge_logo.png</normaloff>:/image/edge_logo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>506</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(152, 206, 142);
	border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image_button/tennis.qrc">
       <normaloff>:/image/chrome_logo.png</normaloff>:/image/chrome_logo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image_button/tennis.qrc"/>
 </resources>
 <connections/>
</ui>
